posthog/constants.py

```diff
@@ -144,6 +146,11 @@ class FunnelVizType(str, Enum):
     STEPS = "steps"
 
 
+class FunnelCorrelationType(str, Enum):
```

Comment:
This seems very specific to the correlation endpoint, is there any reason we wouldn't put it along side the view handler for this? Or the Mixin?

Reply:
Happy to go with convention. I would say there is an obvious upside to colocating, where as I'm not sure there is much of one partitioning by technical type.